packages/react-instantsearch-dom-maps/src/Provider.js

```diff
@@ -61,10 +61,17 @@ class Provider extends Component {
   createBoundingBoxFromHits(hits) {
     const { google } = this.props;
 
-    const latLngBounds = hits.reduce(
-      (acc, hit) => acc.extend(hit._geoloc),
-      new google.maps.LatLngBounds()
-    );
+    const latLngBounds = hits.reduce((acc, hit) => {
+      if (
+        !hit._geoloc ||
+        (Array.isArray(hit._geoloc) && hit._geoloc.lenght === 0)
+      ) {
+        return acc.extend({});
+      }
+
+      const _geoloc = Array.isArray(hit._geoloc) ? hit._geoloc[0] : hit._geoloc;
+      return acc.extend(_geoloc);
```

Comment:
```suggestion
      const geoLocs = Array.isArray(hit._geoloc) ? hit._geoloc : [hit._geoloc];

      return geoLocs.reduce((bounds, current) => bounds.extend(current), acc);
```

I think this will be more correct, can you check if it behaves as you expect for you?

Reply:
Thanks! It's this that I want

Reply:
perfect, I'll let you edit the pull request, and it would also be good if you  could add a test for this, thanks!